config/locales/translation_ja.yml

```diff
@@ -98,6 +98,9 @@ ja:
         overview: 概要
         staff_course_notes: スタッフのコースノート
         status: 状態
+        status_archived: アーカイブ
```

Comment:
For a while, translations for attribute values (not attribute names themselves) are better to locate under activrecords.others.
So in this case, put these status_* under activrecords.others.course.status.* in YAML file.

See also comment in issue #52

Reply:
:ok_woman: